src/mumble/main.cpp

```diff
@@ -3,7 +3,9 @@
 // that can be found in the LICENSE file at the root of the
 // Mumble source tree or at <https://www.mumble.info/LICENSE>.
 
-#include "Overlay.h"
+#ifdef USE_OVERLAY
+	#include "Overlay.h"
```

Comment:
```suggestion
# include "Overlay.h"
```

Reply:
why? Imo this style of indenting is ridiculous and doesn't really contribute to readability, os it could just as well be left out :thinking:

Reply:
See for example:
https://github.com/mumble-voip/mumble/blob/c6b47d114cd66fc271529b0637062b0871384411/src/win.h#L25-L40

Using tabs for indentation would push the the code to the right of the screen.

Reply:
We're using tabs everywhere else, os why are Preprocessor-statements different?

Reply:
@mkrautz came up with the idea.

If I recall correctly he explained the preprocessor style in a review he made for one of the pull requests I opened a few years ago.

Reply:
But I guess you don't happen to know which one that might be?